Description: this subtest requires tap ≥ 14
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-08-18

--- a/test/collect.js
+++ b/test/collect.js
@@ -2,22 +2,6 @@
 const t = require('tap')
 const MP = require('../index.js')
 
-t.test('basic', async t => {
-  const mp = new MP()
-  let i = 5
-  const interval = setInterval(() => {
-    if (i --> 0)
-      mp.write('foo\n')
-    else {
-      clearInterval(interval)
-      mp.end()
-    }
-  })
-  mp.setEncoding('utf8')
-  const all = await mp.collect()
-  t.same(all, ['foo\n','foo\n','foo\n','foo\n','foo\n'])
-})
-
 t.test('error', async t => {
   const mp = new MP()
   const poop = new Error('poop')
